src/class/object_columns.php

```diff
@@ -397,6 +406,24 @@ function prepareData ($zData = '', $sView = 'list')
             $zData['form_type_']       = lovd_describeFormType($zData) . '<BR>' . $zData['form_type'];
             $zData['public_add_']      = '<IMG src="gfx/mark_' . $zData['public_add'] . '.png" alt="" width="11" height="11">';
             $zData['allow_count_all_'] = '<IMG src="gfx/mark_' . $zData['allow_count_all'] . '.png" alt="" width="11" height="11">';
+
+
+            if ($zData['category'] == 'VariantOnTranscript') {
+                // Show genes for which this column is activated.
+                $this->aColumnsViewEntry['related_genes'] = 'Column activated for genes';
+
+                $aGenes = explode(',', $zData['geneids']);
+                $zData['related_genes'] = LOVD_Object::lovd_getObjectLinksHTML($aGenes, 'genes/%s');
```

Comment:
The advantages of using $this->lovd_getObjectLinksHTML() are slightly higher, please update the LOVD_Object::lovd_getObjectLinksHTML() calls to using $this.